ci/prow/e2e-hub-spoke-incluster-build

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+#
+set -euxo pipefail
+
+readonly SKIP_MAKE_DEPLOY="${SKIP_MAKE_DEPLOY:-false}"
+readonly SKIP_ACM_INSTALLATION="${SKIP_ACM_INSTALLATION:-false}"
+
+export CLUSTER_NAME=local-cluster
+export OPERATOR_NAMESPACE=openshift-kmm
+
+
+if [ "$SKIP_ACM_INSTALLATION" == true ]; then
+  echo "Skipping ACM installation"
+else
+  echo "Install ACM..."
+  oc create namespace advanced-cluster-management
+  oc apply -f ci/acm_subscription.yaml
+  timeout 1m bash -c \
+      'until oc get crds -o json | jq -er ".items[].metadata.name | select(.? | match(\"multiclusterhubs\"))"; do sleep 1; done'
+  oc apply -f ci/acm_multiclusterhub.yaml
+  acm_phase=$(oc -n advanced-cluster-management get multiclusterhub/multiclusterhub -o json | jq -r '.status.phase')
+  timeout 30m bash -c 'until [[ ${acm_phase} == "Running" ]]; do sleep 1; done'
+fi
+
+if [ "$SKIP_MAKE_DEPLOY" == true ]; then
+  echo "Skipping KMM deployment"
+else
+  echo "Deploy KMMO..."
+  export KUSTOMIZE_CONFIG_HUB_DEFAULT=ci/install-ci-hub
```

Comment:
```suggestion
  export KUSTOMIZE_CONFIG_HUB_DEFAULT=config/default-hub
```

Reply:
`config/default-hub` will install in `openshift-kmm-hub`, `ci/install-ci-hub` will override it to be installed in `openshift-kmm` as mentioned in a previous comment.